backend/config/curation-api.yml

```diff
@@ -355,6 +355,8 @@ paths:
         request body; if an existing Dataset has this tag, the existing Dataset SHALL be replaced, otherwise a new
         Dataset will be added. MAY include the **id** of an existing Dataset, in which case the existing Dataset
         SHALL be replaced.
+
+        Only presigned AWS S3 URLs and Dropbox shared file links are supported.
```

Comment:
question, non-blocking: do you think its worth noting here for users that they can get AWS S3 URLs using the curation API via the datasets/{dataset_id}/assets endpoint?

Reply:
At the moment if they were to download one of those datasets and try uploading it, it would fail. We add information to the data that our processing pipeline doesn't expect. I don't think encouraging them to do that would be wise.

Reply:
ah right right, makes sense